src/components/Form.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
+
+export default function AddBook() {
+  const dispatch = useDispatch();
+  const [book, setBook] = useState({ title: '', author: '' });
+
+  const handleChange = (event) => {
+    setBook({ ...book, [event.target.name]: event.target.value });
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const newBook = { id: Date.now(), ...book };
```

Comment:
- [OPTIONAL] I can suggest you use `uuidv4` library for the ids generator this is better than using `Date.now()`  for more documentation you can visit this link [uuidv4](https://www.npmjs.com/package/uuidv4) 👍🏻